packages/runtime/src/components/builtin/Carousel/Carousel.tsx

```diff
@@ -174,17 +178,19 @@ const Carousel = forwardRef(function Carousel(
       useResponsiveStyle([slideAlignment] as const, ([alignItems = 'center']) => ({ alignItems })),
     ),
   )
+
   const reelClassName = cx(
     useStyle({ display: 'flex', position: 'relative', flexWrap: 'nowrap' }),
     useStyle(
       useResponsiveStyle([gap] as const, ([gap = { value: 0, unit: 'px' }]) => ({
         margin: `0 ${`${-gap.value / 2}${gap.unit}`}`,
-        [`& > .${slideClassName}`]: {
+        [`& > ${classSelector(slideClassName)}`]: {
```

Comment:
Use the [newly minted `classSelector` utility](https://github.com/makeswift/makeswift/pull/941/files#diff-c12ce42256d2481cc5fb39511d1b141a02982f1eec7e67618114c25d474f6299R83) to construct a valid slide selector.